ops/terraform/services/migrator/main.tf

```diff
@@ -20,10 +20,9 @@ locals {
   layer   = "data"
 
   nonsensitive_common_map    = zipmap(data.aws_ssm_parameters_by_path.nonsensitive_common.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive_common.values))
-  nonsensitive_common_config = { for key, value in local.nonsensitive_common_map : split("/", key)[5] => value }
+  nonsensitive_common_config = { for key, value in local.nonsensitive_common_map : basename(key) => value }
```

Comment:
`basename` seems to be a safer/cleaner way to access the last value of a path. We should consider using this in more places. I recall we recently had an issue due to an incorrect hardcoded index so it would be nice to prevent those.

Reply:
Correct! The initial implementation of our SSM-centered configuration was to be frustratingly opinionated, mandating all _valid_ keys would have the same, specific prefix length. That's all changed now. `basename` works!